.ci/deps.r.sh

```diff
@@ -2,7 +2,7 @@ set -e
 set -x
 
 # R commands
-echo '.libPaths( c( "'"$R_LIB_USER"'", .libPaths()) )' >> .Rprofile
-echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' >> .Rprofile
+echo '.libPaths( c( "'"$R_LIB_USER"'", .libPaths()) )' >>.Rprofile
```

Comment:
hmm. I find this style very jarring.
can it be disabled?
@yukiisbored your opinion on this rule?

Reply:
I don't have much knowledge about these `R` stuff.

Reply:
Can you find a rule to ignore these changes?

Reply:
Understood.
Actually, this was not done by `shellcheck` but by the `shfmt`. 
> can it be disabled?

I can revert this changes. 😃